docs/src/main/asciidoc/qute-reference.adoc

```diff
@@ -374,8 +376,9 @@ The output will be:
 [[if_section]]
 ===== If Section
 
-A basic control flow section.
-The simplest possible version accepts a single parameter and renders the content if it's evaluated to `true` (or `Boolean.TRUE`).
+The `if` section represents a basic control flow section.
+The simplest possible version accepts a single parameter and renders the content if the condition is evaluated to `true`.
+A condition without an operator evaluates to `true` if the value is not considered `falsy`, i.e. if the value is not `null`, `false`, an empty collection, an empty map, an empty array, an empty string/char sequence or a number equal to zero.
```

Comment:
This text is repeated twice, perhaps better to factor it out to a separate section?

Reply:
Hm, you're right. Maybe we could just link this section from the ternary operator section...